nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
+1. Is Nomad deployed single region or multi-region? What is the deployment strategy? How is the high availability and failover designed? How is leader and follower selected?
+1. Is Nomad integrated with consul? How the DNS resolutions works to connect to the consul cluster?
+1. How is Nomad to Consul connectivity? Is it over HTTP secured with TLS? And is there any consul token to encrypt the traffic?
+1. Is the replication happens between the Nomad cluster in different regions?
```

Comment:
Please clarify, is this question asking if Nomad is configured in a multi-region setup?

Reply:
Yes Lars , we want to understand their existing platform if they have nomad clusters running in multi-region or not ? As they are running the nomad cluster on GCP platform , we want to understand their architecture and how the failover is designed .